step-templates/k8s-deploy-kustomize.json

```diff
@@ -0,0 +1,58 @@
+{
+  "Id": "4b0be8ea-e191-4e27-9294-6cc56daaa488",
+  "Name": "Kubernetes - kustomize template deployment",
+  "Description": "Load package with kubernetes configuration and deploy to cluster with kustomize evaluation.",
+  "ActionType": "Octopus.KubernetesRunScript",
+  "Version": 1,
+  "CommunityActionTemplateId": null,
+  "Packages": [
+    {
+      "Id": "df0b9b7f-458e-4ea2-a8e6-1636d0aa7042",
+      "Name": "K8sKustomizeDeploymentConfiguration",
+      "PackageId": null,
+      "FeedId": "feeds-builtin",
+      "AcquisitionLocation": "Server",
+      "Properties": {
+        "Extract": "True",
+        "SelectionMode": "deferred",
+        "PackageParameterName": "K8sKustomizeDeploymentConfiguration",
```

Comment:
We recommend parameter names are "namespaced" with `.` e.g. 

`K8sKustomize.DeploymentConfiguration`